fonts.css

```diff
@@ -73,3 +73,59 @@
   font-size: 20px;
   line-height: 24px;
 }
+
+.card__heading {
+ font-family: Montserrat;
+ font-style: normal;
+ font-weight: bold;
+ font-size: 40px;
+ line-height: 41px;
+}
+
+.card__subheading {
+ font-family: Montserrat;
+ font-style: normal;
+ font-weight: bold;
+ font-size: 16px;
+ line-height: 16px;
+ letter-spacing: 0.07em;
+ opacity: 0.5;
+}
+.text_font_color {
+ font-family: Montserrat;
+ font-style: normal;
+ font-weight: normal;
+ font-size: 15px;
+ line-height: 21px;
+}
+
+.price {
+ font-family: Montserrat;
+ font-style: normal;
+ font-weight: bold;
+ font-size: 40px;
+ line-height: 56px;
+}
+.card__choice-button {
+ font-family: Montserrat;
+ font-style: normal;
+ font-weight: 500;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/itog/pullRequest?prid=5642160)